modin/data_management/factories.py

```diff
@@ -29,6 +27,15 @@ class BaseFactory(object):
 
     io_cls = None  # The module where the I/O functionality exists.
 
+    @classmethod
+    def fill_io_cls(cls):
+        """
```

Comment:
Can we add this comment to the child class? That is what I meant by my comment below, sorry for confusion.

Reply:
I think I've addressed this commend with https://github.com/modin-project/modin/pull/1601/commits/8e204b8dc523866bde5e989f56dfa946eca52000